charts/sda-svc/templates/api-secrets.yaml

```
@@ -36,6 +36,9 @@ stringData:
       verifyPeer: {{ .Values.global.broker.verifyPeer }}
     {{- end }}
       vhost: {{ include "brokerVhost" . }}
+    grpc:
+      host: {{ required "A valid grpc host is required" .Values.global.reencrypt.host }}
+      port: {{ .Values.global.reencrypt.port }}
```

This should be placed after the `db`block so that is is in the correct alphabetical place.

The port is not mandatory since we have  default values in the app config. As such it should be enclosed in a check and only applied if the value is set.

```suggestion
    grpc:
      host: {{ required "A valid grpc host is required" .Values.global.reencrypt.host }}
      {{- if .Values.global.reencrypt.port}}
      port: {{ .Values.global.reencrypt.port }}
      {{- end }}
```